src/phantom.js

```diff
@@ -82,7 +82,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
             }
             networkRequest.changeUrl(url);
         } else if (ignoreRequests.test(url)) {
-            networkRequest.abort();
+            networkRequest.cancel();
```

Comment:
@mikelambert @RyanZim: please check again. I pushed another patch that doesn't throw.

There might be another way to do all this but for now this should do the job as long as there are not side effects.